web/beacon-app/src/features/projects/components/ProjectsTable.tsx

```diff
@@ -41,6 +41,31 @@ const ProjectsTable: React.FC<ProjectTableProps> = ({ projects, isLoading = fals
           return status || '---';
         },
       },
+      {
+        Header: t`Active Topics`,
```

Comment:
At the moment, the `Active Topics` data is not appearing in the table. @pdeziel is working on fixing this.